docker/maven_build/Dockerfile

```diff
@@ -6,6 +6,10 @@ FROM maven:3.8-jdk-11-slim
 
 WORKDIR /tmp
 
+# Registry Common jars
+ADD pom/registry_common/pom.xml /tmp/
+RUN mvn verify --fail-never && mvn clean && rm -rf /tmp/pom.xml
```

Comment:
@tdddblog one other question: what is the reason for adding the `--fail-never` flag? if `mvn verify` fails, shouldn't this break everything else?

Reply:
@jordanpadams We just want to cache (download) all the dependencies. Build will fail because there are no sources.